archetypes/Exchange/Summary/ApproxCommitFee.tsx

```diff
@@ -3,20 +3,17 @@ import BigNumber from 'bignumber.js';
 import { toApproxCurrency } from '~/utils/converters';
 import { Transparent } from './styles';
 
-export const ApproxCommitGasFee: React.FC<{
-    amount: BigNumber;
-    gasFee: BigNumber;
-}> = ({ amount, gasFee }) => {
+export const ApproxCommitFee = ({ amount, fee }: { amount: BigNumber; fee: BigNumber }): JSX.Element => {
     const getFee = () => {
         if (amount.toNumber() === 0) {
```

Comment:
this doesn't seem to be getting hit despite fees being 0 (only when there is a value in the amount input)

no value:
![image](https://user-images.githubusercontent.com/14157626/178663018-199da56d-710c-4f7e-8dac-5d21a4276328.png)

with value:
![image](https://user-images.githubusercontent.com/14157626/178663073-9dde084c-8912-441e-aeda-cd722c397107.png)